algopy_testing/src/algopy_testing/op.py

```diff
@@ -568,6 +568,47 @@ def json_object(a: Bytes | bytes, b: Bytes | bytes, /) -> Bytes:
         return Bytes(result_string.encode())
 
 
+class Scratch:
+    @staticmethod
+    def load_bytes(a: UInt64 | int, /) -> Bytes:
+        from algopy_testing import get_test_context
+
+        context = get_test_context()
+        slot_content = context._scratch_space[a]
+        match slot_content:
+            case Bytes():
+                return slot_content
+            case bytes():
+                return Bytes(slot_content)
+            case UInt64() | int():
+                return itob(slot_content)
```

Comment:
If the stored value is the wrong stack type it should be an error, same with load_uint64